packages/webapp/src/Repo.tsx

```diff
@@ -256,6 +272,22 @@ export const Repo: React.FC<Props> = ({
           </>
         )}
       </td>
+      <td>
+        {data.lastDatapoint.sonarCloud.testCoverage ? (
+          <span
+            className="test-coverage"
+            style={sonarCloudTestCoverageStyle(
+              data.lastDatapoint.sonarCloud.testCoverage,
+            )}
+          >
+            {data.lastDatapoint.sonarCloud.testCoverage}
+          </span>
+        ) : (
+          <span style={{ color: "var(--color-success)" }}>
```

Comment:
nit: Should we just avoid setting the style when test coverage is missing? Default text color would then be black, no?

Reply:
Default is black. The default black is also used when Snyk data is lacking. I will remove the style-block.